OpenMarket/OpenMarket/Model/Network/ImageEncoder.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ImageEncoder.swift
+//  OpenMarket
+//
+//  Created by 우롱차, Donnie on 2022/06/01.
+//
+
+import UIKit
+
+struct ImageEncoder {
+    private enum Constant {
+        static let defaultMaximumImageSize = 300 * 1024
+        static let defualtResizeWidthValue: CGFloat = 100
+        static let defaultCompressionQualityValue: CGFloat = 1
+    }
+    private let maximumImageSize: Int
+    private let resizeWidthValue: CGFloat
+    private let compressionQualityValue: CGFloat
+    
+    init(maximumImageSize: Int = Constant.defaultMaximumImageSize
+, resizeWidthValue: CGFloat = Constant.defualtResizeWidthValue, compressionQualityValue: CGFloat = Constant.defaultCompressionQualityValue) {
+        self.maximumImageSize = maximumImageSize
+        self.resizeWidthValue = resizeWidthValue
+        self.compressionQualityValue = compressionQualityValue
+    }
+    
+    func encodeImage(image: UIImage) throws -> Data {
+        
+        var appendedData = Data()
+        
+        guard let imageData = image.jpegData(compressionQuality: compressionQualityValue) else {
+            throw(UseCaseError.imageError)
+        }
```

Comment:
개인마다 다를 수 있지만, 일반적으로 소괄호를 사용하지 않고 작성합니다 🤔 
```suggestion
        guard let imageData = image.jpegData(compressionQuality: compressionQualityValue) else {
            throw UseCaseError.imageError
        }
```